prebid-mobile/pbm-api/ios/pbm-plugin-renderer.md

```diff
@@ -174,7 +174,7 @@ public class SampleCustomRenderer: NSObject, PrebidMobilePluginRenderer {
 In addition to this documentation you have samples on hand which can be get from the Prebid Mobile SDK repository:
 
 * [PrebidMobilePluginRenderer](https://github.com/prebid/prebid-mobile-ios/blob/master/PrebidMobile/PrebidMobileRendering/PluginRenderer/PrebidMobilePluginRenderer.swift)
-* [PluginEventDelegate](https://github.com/prebid/prebid-mobile-ios/blob/master/PrebidMobile/PrebidMobileRendering/PluginRenderer/PluginEventListener.swift)
+* [PluginEventDelegate](https://github.com/prebid/prebid-mobile-ios/blob/master/PrebidMobile/PrebidMobileRendering/PluginRenderer/PluginEventDelegate.swift)
```

Comment:
[PluginEventDelegate](https://github.com/prebid/prebid-mobile-ios/blob/master/PrebidMobile/PrebidMobileRendering/PluginRenderer/PluginEventDelegate.swift) does not exist on master.

Current class name is still [PluginEventListener](https://github.com/prebid/prebid-mobile-ios/blob/master/PrebidMobile/PrebidMobileRendering/PluginRenderer/PluginEventListener.swift)

An update must to be added to prebid ios sdk

Reply:
Ok, PR opened 
https://github.com/prebid/prebid-mobile-ios/pull/1046